src/app/api/chat+api.ts

```
@@ -1,39 +1,58 @@
+      role: z.enum(['user', 'assistant', 'system', 'function']).optional(),
+      content: z.string(),
+    })
+  ),
+  modelName: ModelNameSchema.default(FREE_MODELS[0]),
+});
 
-  const result = streamText({
-    model: google('gemini-2.0-flash'),
-    messages,
+// Helper: stream model answer from OpenRouter. Kept generic so can be reused.
+export const askModelStream = (modelName: string, messages: { role?: string; content: string }[]) => {
```

ESLint

[error] Replace `modelName:·string,·messages:·{·role?:·string;·content:·string·}[]` with `⏎··modelName:·string,⏎··messages:·{·role?:·string;·content:·string·}[],⏎`

(prettier/prettier)